notebook/LICENSE.txt

```diff
@@ -0,0 +1,30 @@
+Copyright (c) 2005-2023, NumPy Developers.
```

Comment:
NOTE: we will need to add a README and license to the repo itself as only hvaing this one attributed to the NumPy team might be confusing